odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -349,7 +350,10 @@ Connecting to your virtual machine
 
     $ sudo ./script_to_run.sh
 
-  The script will ask you for the server's domain and an administration email address to configure https on the server.
+  The script will ask you for the server's domain and some questions (as shown in the picture below) along with an administration email address to configure https on the server.
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine1.png
```

Comment:
Can you add red boxes to the key info? I was a bit overwhelmed by the amount on the screen, good to point to what is needed.

Reply:
Sure.